SRC/CLASSES.CPP

```diff
@@ -1237,7 +1237,8 @@ void Level::load( char name[13] )
 {
     unsigned int a, bullets, weapons;
     FILE *dat;
-    dat = fopen( name, "rb" );
+    std::string path = data_path( name );
+    dat = fopen( path.c_str(), "rb" );
```

Comment:
In this case `fopen( data_path( name ).c_str(), "rb" );` would have been ok, because the string's scope is valid until `fopen` returns. :)

https://stackoverflow.com/questions/10006891/stdstringc-str-and-temporaries

But `const char* str = data_path( name ).c_str();` is not safe because the scope ends before using the pointer.

No need to change for me though. Only tweak I would have done is to use `const std::string` instead.

Reply:
Nice! I changed all places to use temporaries :)